resources/eventing/values.yaml

```diff
@@ -39,7 +39,7 @@ authentication:
   # bebNamespace is the name of the namespace BEB is provisioned with
   bebNamespace: ""
 
-preUpgradeJob:
-  image:
-    name: eu.gcr.io/kyma-project/tpi/k8s-tools
-    tag: "20210610-d25e85b1"
+#preUpgradeJob:
```

Comment:
Do you need these? Should it be removed?

Reply:
Not sure, I have put it back.

Reply:
its also in the release-1.24 branch. But I am unable to find its usage anywhere in the helm charts.